### ISSUE TITLE
Different Learning Rate For Different Model Parameters

### ISSUE THREAD

**Issue by guru4elephant** [PaddlePaddle/Paddle#8104]
I'm trying to set different learning rates for different parts of model parameters. Is there any example for this purpose?

**Comment by ranqiu92** [comment]
如下代码应该就可以，相应参数的实际学习率=全局学习率 * 此处设置的学习率（即0.1）
```python
fc = fluid.layers.fc(input=input, param_attr=ParamAttr(learning_rate=0.1))
```
